src/nvim/ops.c

```diff
@@ -2361,11 +2324,29 @@ static void free_yank_all(void)
  *
  * Return FAIL for failure, OK otherwise.
  */
-int op_yank(oparg_T *oap, int deleting, int mess)
+int op_yank(oparg_T *oap, bool mess) {
```

Comment:
Is it a generic refactor that `int`/`OK`/`FAIL` should become stdbool ? Then perhaps it should be added to the style guide. (Right now it says `int` should be used for "error codes")